### PR TITLE
Increase build timeout pipeline

### PR DESCRIPTION
**Pull request by see-quick**
This PR increases the timeout for the build pipeline. It seems that we can hit a problem where timeout can occur like we hit it in the main branch resulting in not pushing the images. 

This problem does not occur on the branch build because we do not push and tag them. :)

### CODE REVIEW

**Review by Frawless — APPROVED** [review]

**Review by im-konge — APPROVED** [review]

**Review by jankalinic — APPROVED** [review]

**Review by scholzj — COMMENTED** [review]
I guess you should explain why does it really need to take this long as I think it is not really clear.

**Comment by scholzj** [comment]
Is it because you seem to be building too many of Kafka versions? If so, where exactly are they used? AFAIK we use only one version. So it likely makes sense to cut the Kafka versions in the first place?

**Comment by see-quick** [comment]
> Is it because you seem to be building too many Kafka versions? If so, where exactly are they used? AFAIK we use only one version. So it likely makes sense to cut the Kafka versions in the first place?

Well, there have been many discussions about which Kafka versions to support. We decided to always support every major version with its latest minor (e.g., for 3.5, it would be 3.5.1), and so forth. I'm just adhering to this established pattern. If we want to adjust this approach to support as many Kafka versions as possible without exceeding a 60-minute limit, I'm fine with removing some, like 2.8.2 or others.

**Comment by scholzj** [comment]
Well, if you ignore the problems, you will be increasing it to 120 minutes in few months. As far as I know, we do not use it anywhere and @tombentley who was the main proponent of these old versions doesn't use the Test Container at all in his code. So isn't it easier and better to just drop a bunch of the old versions?

**Comment by see-quick** [comment]
Hearing nothing from @tombentley I think we can proceed with the drop of the old Kafka version @scholzj ? :)

**Comment by scholzj** [comment]
@see-quick I guess if you would want to try more conservative approach, you can also try to use a shared base image first for example. As you right now seem to run `microdnf update     && microdnf install java-11-openjdk-headless shadow-utils     && microdnf clean all` 34 times while it would be needed only 4 times (once per platform). Depending on how many Kafka versions you remove, this would be significant speedup as well.

File: images/base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN microdnf update \
+    && microdnf install java-11-openjdk-headless shadow-utils \
```
Comment (scholzj) [inline images/base/Dockerfile:6]:
```suggestion
    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-11-openjdk-headless shadow-utils \
```

I think this wouldmake the image a bit smaller and with less CVEs.

**Review by scholzj — APPROVED** [review]

**Comment by see-quick** [comment]
So with your suggestion @scholzj, I managed to speed up the building process from 59 minutes to just 17 minutes :) Thanks for it 👍

**Pull request merged**